app/src/main/java/org/wikipedia/createaccount/CreateAccountActivity.kt

```diff
@@ -319,6 +324,8 @@ class CreateAccountActivity : BaseActivity() {
                 return ValidateResult.INVALID_USERNAME
             } else if (password.length < PASSWORD_MIN_LENGTH) {
                 return ValidateResult.PASSWORD_TOO_SHORT
+            } else if (password.toString() == username.toString()) {
```

Comment:
Let's make the comparison case-insensitive.

Reply:
Yes that's a good idea!